### ISSUE TITLE
zwavejs2mqtt not working in lxc

### ISSUE THREAD

**Issue by usrstephan** [zwave-js/zwavejs2mqtt#1944]
I used zwave2mqtt for years in an lxc under proxmox ve with usb-passthrough and had no issues.
Now I changed to zwavejs2mqtt and there seems to be a permissions issue to connect to the zwave-adapter.
Does anyone else have the same problem/a solution?

**Comment by kpine** [comment]
Did you read this? https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/app_crash?id=application-does-not-start-after-updating-to-version-510x-or-higher

**Comment by usrstephan** [comment]
yes I did. It claims: "Driver: Failed to open the serial port: Error: Permission denied, cannot open /dev/serial...."
The strange thing is everything works on a vm but not in a lxc - that's why I asked for this issue.
short addition: in pve it is possible to map serial devices to specific device names inside the container. so I tried different ways before raising this topic.
the listing of ls -l shows full permissions inside the container but they get completely lost after starting zwavejs2mqtt (from 777 to 000).

**Comment by kpine** [comment]
How about the part where it says:

> If this does not help, the stick restart can be disabled by disabling Soft Reset option under Zwave settings, but this may limit functionality.

Did you try disabling soft-reset to see if it makes a difference?

It seems you also didn't fill in the issue template which [requests logs](https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/bug_report). Without logs it's hard to assist. It could be the soft-reset, or it could be something else.

**Comment by usrstephan** [comment]
the soft-reset actually made the difference - now it works.
Thanks very much!!!
What are the restrictions of disabling soft-reset?
Are you interested in extending the getting started section for lxc under proxmox (I could send you mine by mail)

**Comment by blhoward2** [comment]
Without soft reset only a few things don't work, mainly changing the region or backing up or restoring the stick. 

You're welcome to PR changes to the docs if you want to suggest any. They're in the repo under docs like all of the other code.

**Comment by robertsLando** [comment]
Closing this for now as it's not a bug with application

**Comment by mvonrose** [comment]
I just installed zwavejs2mqtt for the second time, and this issue is coming up on my new install. I installed, as I did the first time, by cloning the github repo and build/starting it with yarn. Error on opening the serial port and that permissions are denied. I followed this post: https://zwave-js.github.io/zwavejs2mqtt/#/troubleshooting/app_crash?id=application-does-not-start-after-updating-to-version-510x-or-higher, and still nothing. Is anyone having this pop up again recently?
